docs/de/trouble/install_fixer.MD

```diff
@@ -0,0 +1,72 @@
+---
+title:       "ioBroker Installation Fixer"
+lastChanged: "13.06.2019"
+---
+
+# Probleme mit Benutzerrechten der ioBroker Installation beheben
+
+Der Installations Fixer löst Probleme mit den Benutzerrechten der ioBroker installation.
+Ende 2018 und vor allem im Januar 2019 hat @AlCalzone den ioBroker Installer komplett überarbeitet und damit gehören in den inzwischen aktuellen Version von Anfang Februar 2019 auch alle Rechteprobleme der Vergangenheit. ioBroker läuft damit nicht mehr als "root" sondern unter einem eigenen User der alles darf was er für die aktuellen ioBroker Adapter können muss.
+Für alle neuen Installation ist also alles bestens.
+
+Was ist aber wenn jemand ioBroker früher installiert hat? Er noch als root läuft? Oder in den ersten Tagen der neuen Installationsroutine?
+**Auch dafür haben wir jetzt dank @AlCalzone eine Lösung: den Installation-Fixer**
+
+Mittels einem Kommando wird eine bestehende Installation in /opt/iobroker auf den gleichen Stand gebracht wie eine aktuelle neue Installation. Das Skript kann auch in Zukunft immer wieder genutzt werden um die Installation diesbezüglich zu Aktualisieren. 
+Wichtig: Diese Skript aktualisiert weder nodejs, npm noch js-controller oder irgend einen Adapter. Nur die Systemrechte u.a. werden bearbeitet.
+Versucht es und gebt Feedback im Diskussionsthread unter https://forum.iobroker.net/topic/20212/diskussion-zum-neuen-installation-fixer
+
+**%(#ff0000)[Bitte beachten]**:  Anwendung unter Docker sollte, weil eh alles als root läuft, nicht nötig sein und wir raten aktuell mangels klarer Erfahrungen und Feedback von einer Anwendung ab. Falls es doch jemand versuchen will und Feedback geben will: Anwendung komplett auf eigene Gefahr. Unbedingt vorher ein Backup machen und wissen was man tut!
```

Comment:
I did not implemented all possible markdown features. So the color is not supported.
Please use `!> ` 
Like here: https://github.com/ioBroker/ioBroker.docs/blob/master/docs/de/README.md#L7
And you will get:
![image](https://user-images.githubusercontent.com/4582016/59429320-d9a1e500-8ddf-11e9-8089-4d61d86dca57.png)

Reply:
OK changed it. 
Color would be nice, makes it easier to transport Forum posts to docs.